src/adapters/udp.rs

```diff
@@ -231,6 +324,66 @@ impl Local for LocalResource {
         }
     }
 
+    #[cfg(target_os = "linux")]
+    fn accept(&self, mut accept_remote: impl FnMut(AcceptedType<'_, Self::Remote>)) {
+        let mut input_buffer = vec![0; MAX_LOCAL_PAYLOAD_LEN];
```

Comment:
Do we really need this to be a vector? Could it be an array to avoid allocations?